keyboards/keebsforall/freebird75/info.json

```
@@ -0,0 +1,463 @@
+{
+    "manufacturer": "KeebsForAll",
+    "keyboard_name": "freebird75",
+    "maintainer": "Elliot Powell",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D0", "D1", "D2", "F7", "F6", "F5", "F4", "F1", "F0", "E6", "B0", "B1", "B2", "B3", "B7"],
+        "rows": ["C7", "C6", "B6", "B5", "B4", "D7"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x7575",
+        "vid": "0x6B66"
+    },
+    "layouts": {
+        "LAYOUT_75": {
```

This should match one of the community layouts, or should be left as just `LAYOUT`.
```suggestion
        "LAYOUT": {
```

Either way, that necessitates a change, and needs to be applied to the keymap.c files, as well.